client/src/util/testScheduler.ts

```diff
@@ -160,9 +166,12 @@ export const filterHelperTimeOverlaps = (combination: CourseSection[]): boolean
     for (let i = 0; i < combination.length; i++) {
         for (let j = i + 1; j < combination.length; j++) {
             let firstCourseTimeExists: boolean = combination[i]["starttime"] !== undefined && combination[i]["endtime"] !== undefined;
-            let secondCourseTimeExists: boolean = combination[i]["starttime"] !== undefined && combination[i]["endtime"] !== undefined;
+            let secondCourseTimeExists: boolean = combination[j]["starttime"] !== undefined && combination[j]["endtime"] !== undefined;
+            const firstDay = combination[i].day;
+            const secondDay = combination[j].day;
 
             if (firstCourseTimeExists && secondCourseTimeExists) {
+                if (firstDay !== secondDay) continue;
```

Comment:
Is this just an optimization to skip if they aren't on the same day, because in that case we wouldn't need to check if they overlap?

Reply:
I noticed the case where we have `MFT 10 AM` and `TTh 10 AM` classes getting filtered out, not sure if we take care of that case below -- noticed that we're using `05-17-2018` for all courses is that intended?

Reply:
Oh I see. Right, I totally just disregarded days - oops :D, thanks for fixing that. `5-17-2018` is totally arbitrary since these date libraries need an actual date to be passed in.

Reply:
There might be a way to only pass times in, but I havent looked too deeply into dayjs library.